tests/selectors.js

```diff
@@ -53,13 +53,19 @@ describe( 'uncss', function () {
         expect( rawcss ).to.have.length.above( 0 );
     });
 
-    it( 'should read .uncssrc files', function () {
+    it( 'should read .uncssrc files', function (done) {
         uncss( readFile('index.html'), {
+            csspath: 'tests',
+            report: true,
             uncssrc: path.normalize('tests/.uncssrc')
         }, function ( err, res, report ) {
+            if( err ) {
```

Comment:
Yup, and it's why I want to switch to ESLint but haven't had the time yet :/

If you are up to it, feel free to make a PR against master using https://github.com/mpc-hc/mpc-hc.org/blob/master/.eslintrc.json and adapting it.

Thanks!